src/main/java/uk/gov/hmcts/reform/ccd/documentam/model/GenerateHashCodeResponse.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.ccd.documentam.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class GenerateHashCodeResponse {
```

Comment:
Picky but would this be better as `GeneratedHashCodeResponse`

Reply:
👍  - have renamed